openaerostruct/geometry/geometry_mesh.py

```diff
@@ -79,10 +79,15 @@ def setup(self):
         val = np.ones(ny)
         if "chord_cp" in surface:
             promotes = ["chord"]
+            if "chord_scaling_pos" in surface:
+                gamma = surface["chord_scaling_pos"]

```

Comment:
Once you change the option name, also change this variable name. Also can't this if else block be moved outside of the `if "chord_cp" in surface` block so that you don't need to duplicate the `gamma = 0.25` line? If you want to be even nicer, you could throw a warning if `"chord_scaling_pos"` is specified but there are no chord DVs.